### DISCUSSION TITLE
Adding a link to the networking lesson that's currently in 'Getting Hired' earlier in both paths

### DISCUSSION THREAD

**Discussion by JustWaveThings** [TheOdinProject/curriculum#28760] [Ideas]
Is there any appetite to add a link with a small blurb at the end of React/Beginning of Node (JS path. Not sure where to put it on the Ruby path)  to the Getting Hired lesson on '[Networking](https://www.theodinproject.com/lessons/getting-hired-professional-networking)'.  It could be beneficial to explicitly bring it up at that point as that anecdotally seems to be the point where people start to think about applying/start applying?

**Comment by jwaddell10** [comment]
I just had this same idea and was about to create an issue. To add to it, most people seem to be getting hired (at least per the success stories in discord) before completing the curriculum. Moreover, many folks who are hired say it was because of someone in their network that they got the job. Professional/personal networks take time to build, so I think it would be helpful to introduce networking earlier on in the curriculum, I second @JustWaveThings suggestion of putting it at the beginning of NodeJs
